Products/ZenModel/Device.py

```diff
@@ -1124,7 +1124,7 @@ def updateDevice(self,**kwargs):
 
         # apply any zProperties to self
         for prop, value in zProperties.items():
-            if value is not None:
+            if value is not None and value != "":
```

Comment:
If `value` is always a string (or `None`), this can be shortened to
```
    if value:
        ...
```
because `None` and empty strings are considered False in boolean expressions.